docs/DeveloperGuide.md

```diff
@@ -281,47 +296,269 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 *{More to be added}*
 
+-->
+
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `RC4HDB` and the **Actor** is the `user`, unless specified otherwise)
+
+
+**Use case: Getting help with the application**
+
+**MSS**
+1. User needs help in utilizing RC4HDB.
+2. User enters the help command.
+3. RC4HDB displays a message that directs the user to our user guide.
+   
+    Use case ends.
+
+**Use case: Adding a single resident**
+
+**MSS**
+1. New resident moves into RC4.
+2. User gets personal details of the resident.
+3. User enters the add command and enters the resident information in the required format.
```

Comment:
Based on W7.1c Requirements → Specifying Requirements → Use Cases → Details, we should give the intention of the actor, and not the mechanics. 

I recommend replacing this step with "User enters the add command." instead :)